### DISCUSSION TITLE
Historical continent data

### DISCUSSION THREAD

**Discussion by MalQaniss** [owid/covid-19-data#2584] [General]
What dataset should I use to get a historical continent data?

**Comment by lucasrodes** [comment]
Hi @MalQaniss,
As highlighted [in our documentation](https://github.com/owid/covid-19-data/tree/master/public/data), you could use the dataset at https://covid.ourworldindata.org/data/owid-covid-data.csv. In there, you can filter the rows by `location=World` (or `iso_code=OWID_WRL`).
